incident_viewer_demo/incident/incident.go

```diff
@@ -0,0 +1,63 @@
+package incident
+
+import (
+	"strconv"
+	"time"
+)
+
+type Incident interface {
+	goodPeriod() (time.Time, time.Time)
+	badPeriod() (time.Time, time.Time)
+	severity() float64
+	testsAffected() int
+	goodPeriodInfo() string
+	badPeriodInfo() string
+	incidentInfo() string
+}
+
+type DefaultIncident struct {
+	goodStartTime    time.Time
+	goodEndTime      time.Time
+	badStartTime     time.Time
+	badEndTime       time.Time
+	avgGoodDS        float64
+	avgBadDS         float64
+	severityDecimal  float64
+	numTestsAffected int
+}
+
+func (i *DefaultIncident) goodPeriod() (time.Time, time.Time) {
```

Comment:
should these methods be explicitly getters (have a "get" in the method signature)? or does that matter?

Reply:
I removed the 'get's to be consistent with the UML in the README. @roman-rosenast had a reason for doing this right?

Reply:
Yes, will change methods to be consistent with the "getGoodPeriod" style